dev-requirements.txt

```diff
@@ -9,3 +9,4 @@ sphinx_rtd_theme
 pytest-asyncio
 requests
 pytest-timeout
+jsonpath_ng
```

Comment:
This needs to be added to komodo as well 🙂

Reply:
True, thanks!

Reply:
Does it? Aren't these resolved dynamically?

Reply:
Ahh, you are correct! Didn't realise that they were only test requirements!

Reply:
Okay, no changes in komodo then